birdhouse/deprecated-components/project-api/config/proxy/canarie_api_monitoring.py.template

```diff
@@ -0,0 +1,6 @@
+PLATFORMS['pavics']['monitoring']['Project'] = {
+    'request': {
+        'url': 'https://${PAVICS_FQDN_PUBLIC}/project-api/explorer/'
+    }
+}
+PLATFORMS['pavics']['stats']['route'] = '/project-api/.*'
```

Comment:
Copy-paste error? Should be `PLATFORMS['server']`.

Reply:
Not a copy paste error, but I'm seeing that the canarie-api configuration has changed significantly since this PR was opened. Since this PR mostly moves files, git isn't showing any merge conflicts but essentially this PR is now moving out of date files. I will have to re-do a lot of the moved files based off the of the new content of the canarie-api scripts